stdlib/websub/src/main/ballerina/websub/commons.bal

```diff
@@ -526,6 +525,16 @@ public type WebSubHub object {
     # + return - `error` if an error occurred with unregistration
     public function unregisterTopic(string topic) returns error?;
 
+    # Retrieves an array of topics which registered in the Hub.
```

Comment:
Should we say "Retrieves topics currently recognized by the Hub." ?

This is because when topic registration is not required, an implicit registration is done, so saying "registered" could be misleading, WDYT?